content/applications/websites/website/configuration/multi_website.rst

```diff
@@ -52,7 +55,7 @@ top right corner and select the website you want to switch to.
    :alt: Website selector
 
 .. note::
-   When you switch websites, you are redirected to the other website, to the same page (URL) as the
+   When you switch websites, you are :ref:`redirected <website/URL-redirection>` to the other website, to the same page (URL) as the
    current one. If the page you are currently viewing does not exist on the other website, you are
    redirected to a 404 error page. Once redirected, click :guilabel:`Create page` to create the
    page.
```

Comment:
This doesn't seem to be true anymore in 17.0. I may be wrong but it seems we're always redirected to the Homepage now: when I tested in 16.0, when I was on the /shop page and switched to another website, i landed on the /shop page of the other website. In 17.0, I land on the Home page.
Also, I wouldn't add a ref to the redirections part of the Pages doc page. In my understanding, this behavior doesn't quite fit the definition of a traditional redirection, as no redirection record is created and this doesn't concern website visitors. Instead, it's simply how the feature works when administrators switch between websites.
Of course, I could be mistaken, so it might be a good idea to verify this.

Reply:
You're absolutely right. Updated accordingly.